### ISSUE TITLE
General design of The Idea

### ISSUE THREAD

**Issue by towerofnix** [mathducation/mathducation.github.com#1]
The README currently describes the project as this:

>Free maths learning that's based on intuition rather than formulae.

I'm interested to know exactly what you mean by that?

(PS, I'm part of the organization, but I don't seem to have access to label issues, pull PRs, etc.)

**Comment by joker314** [comment]
Uh like I was thinking we could have a tug of war that's completely balanced; and then some people involved have different strength, and we don't know person "x"'s strength. = algebra.

_Shrugs_. If you have a better idea we can do that

**Comment by joker314** [comment]
Okay, you should now be able to do stuff

**Comment by towerofnix** [comment]
Hmmmmm. My way of teaching algebra is probably different from yours :)

I *love* using formulas - well, equations - to show math. But I always want to make sure that when I'm explaining something, I'm explaining *why* it works, and *that* should be intuitive. It makes sense that multiplying by the fraction (1/4)/(1/4) doesn't change the value of whatever you're multiplying it by, because you're literally just multiplying by the number 1 (and obviously 1x = x).

I don't try to *disguise* my math as something that isn't, well, plain old rules we've invented! All of math is based on those rules, so I want to teach them. *However,* I absolutely want to make sure I'm teaching them in a way that makes sense. I can just tell you the multiplicative identity rule, which is that one times any number equals that number (or 1x = x), but I want to make sure you *get* that, and that's where I *explain* the rule, using real-life analogies where they make sense. When we multiply something by one, we're just taking one of that something, right? And then we've just taken the thing we already had. So, if I had a book, and I took one book, then I'd just still have the book. Or if I had a stack of three plates, and I took one stack of three plates, I'd still have a single stack of three plates.

An equation is not a case of two groups of people (which I suppose are an analogy for terms in an equation) in a tug-of-war, and your goal is to keep it balanced. In truth, an equation is two bunches of numbers and variables (well, terms) which are defined as equal to each other, and your goal is (usually) to find a number for which you can replace the variable and still have the equation turn out to be true (so both the expressions should still be equal). *However,* because the analogy is good, we can use it to help teach. For example, let's think about how solving an equation works. I'll solve an example equation; here's a simple one:

---

4x - 3 = 5

Let's think of this as a tug-of-war. There are three people here, and they each have their own strength. On the left side, 4x and -3. On the right side, 5. We already know the strengths of -3 and 5 -- those *are* their strengths. But we don't actually know the strength of 4x. We only know that 4x has quadruple the strength of "x".

(Note that somebody having "negative three" strength is not really confusing to either of us, but *we* understand how equations and negative numbers work. A student new to math equations probably does not, so they would rely on the analogy making sense. And somebody with -3 strength... doesn't really make sense. I suppose you could say there's somebody *betraying* that side of the game/equation, that is, pulling in the opposite direction of the side they're on? Maybe that would work.)

*Generally,* the way to solve a tug-of-war -- that is, to find the strength of x -- is to make the equation simpler and simpler until we only have two people in the game: the one named "x" and whatever the strength of "x" is. Our final, complete equation should look something like this:

x = ?

Of course, the question mark would be an actual number. But we don't know what number goes there yet; that's what we're trying to figure out.

We have to remember that we *always* want to keep the tug-of-war *balanced*; we never want one side to pull harder than the other. So if we do something to get rid of a player on one side, we have to do the same thing to the other side.

We want to get closer to "x = ?". We're currently at "4x - 3 = 5". To get closer, we'll get rid of the "-3" player. But how can we do this while keeping the two sides to have the same total strength? Well, we can't get *rid* of the "-3" player, because we'd also have to get rid of a "-3" player on the other side, and there is no such player. But what we *can* do is add a *new* player, whose strength is PLUS three (well, 3). And we can easily do that to both sides, since we're adding a totally new player, not getting rid one one. So we end up with a tug-of-war equation that looks like this:

4x - 3 + 3 = 5 + 3

Let's take a closer look at the left side, which reads "4x - 3 + 3". So we have three players on that side: 4x, -3, and +3. We know the strengths of -3 and +3: those are just -3 and +3, of course. But now I ask you: What is the *total* strength of the players -3 and +3? What impact do they have, together, on their team?

The answer is zero; since we have two people who have the same strength but in opposite directions, they're actually pulling against each other, so they just cancel out and don't have any effect. This actually means we can *ignore* them, and totally wipe them from the equation, like this:

4x = 5 + 3

(Note: I could have gone with saying something along the lines of "The -3 player is pulling in the opposite direction of left side, so really it's helping the *right* side, and we can move it to the other side, but as a +3, not a -3: 4x = 5 + 3." But I think that the way I explained it is more faithful to how algebra really works, i.e. that generally, performing *any* operation to one side of an equation and *also* applying it to the other side will get you an equation that *looks* different, but *means* the same thing. In this case, our operation was adding a player whose strength is +3 to both sides.)

We're definitely closer to our goal of "x = ?"! Now one side is just 4x. But before we can go any further, we actually have to do a little bit of cleaning up. Maybe when you read our current equation (4x = 5 + 3), you saw what we can change?

Let's look at the *total* strength of each side. Well, we know that for the left side, "4x", we only have one player, and their strength is 4 times the strength of x. So the left side's strength is just 4x. On the right-hand side, "5 + 3", we have two players: 5 and 3. These are just numbers, so we can add their strengths together: 8. So the total strength of the right side is 8. That means we can replace "5 + 3" with "8" in our equation, which makes it look like this:

4x = 8

Wow! Now we're really close to "x = ?". If we read out our equation, we'll see that quadruple the strength x is the same as the other side's strength, 8. So the question is, how do we get from quadruple a strength (in this case, that's shown as 4x) to just the strength itself? Well, it's pretty clear: we divide the strength by 4. So we'd get that "x = 8". But *hold on,* we need to remember that we always have to keep the tug-of-war in *balance*. That means that if we do something to *one* side of the game, we also have to do it to the *other*. This isn't too hard, though - it just means that since we divided the left side by 4, we also have to divide the right side. That means we get this:

x = 2

Wow! This is our final tug-of-war equation, since it reads along "x = ?". And we know what that question mark is now -- the number 2 -- so we know the strength of x!

But how can we be absolutely sure this works? Well, we'd have to actually *try* the game, right? Luckily, we don't need to set up an actual real-life game of tug-of-war (and it would probably be hard to find people with exactly the right strengths, anyways). We can just play out the game from the start using our math equation, but with "x" replaced with 2:

4x - 3 = 5

4(2) - 3 = 5

Well, we already know what the strength of the right side is. That's just five. We just need to figure out what the total strength of the left side, "4(2) - 3", is. Again, we have two players on that side: 4(2) and -3. And now we can actually find out the strength of the first player: 4 times 2 is 8, of course! So we can change our 4(2) player to show 8 instead, and that looks like this:

8 - 3 = 5

Now it's pretty easy to find out the total strength of the left side: just add 8 and -3. And of course, that's just 5. So we'll replace the side "8 - 3" with "5", and here's what we get:

5 = 5

And there we go! Both sides have the same strength, and that means that the tug-of-war is balanced. So we've figured out what strength we can replace "x" with *and* actually made sure that it works.

Remember, when you're actually answering a problem in a workbook, you'll want to write out the equation that tells you what strength "x" is. So you'd answer that "x = 2", here.

---

This is actually very similar to how we normally solve equations - that is, without thinking of it as a game of tug-of-war. In fact, we took *exactly the same* steps! If you've ever seen a math workbook of someone who's already done algebra, you've probably seen something like this:

> Question 1:
> 4x - 3 = 5
> 4x = 5 + 3
> 4x = 8
> x = 2
>
> Check:
> 4x - 3 = 5
> 4(2) - 3 = 5
> 8 - 3 = 5
> 5 = 5

When we normally do math equations, we really are doing the same thing, but basically with different words. Instead of saying we have "players" on each side and that these players have "strengths", we say we have "terms" on side, and these terms have "values". And instead of saying we're keeping a tug-of-war balanced, we're really just keeping sides an equation balanced. But that's done exactly the same way!

Now that we understand just how solving an equation works, we'll start talking about equations using words like "terms" and "values" and "expressions". But we won't throw these all at you just like that - we'll *explain* them, while comparing them to our tug-of-war analogy. The important part is that you understand *how* and *why* they work, and while we'll definitely teach and learn using our existing knowledge of math and such, we'll also use analogies to make everything a bit more intuitive.

---

@joker314 SORRY FOR THE ESSAY 💙

**Comment by joker314** [comment]
@towerofnix terms and players; values and strengths -- that's just different notation for the same thing. Algebra doesn't care if we're talking about a tug of war, or as abstract numbers -- so long as they both share the same axioms everything just works. I'm not sure what your point is, you just wrote a brilliant piece that would help students, and would be wonderful in the course; and it's not hiding the beauty of algebra, instead it's showing it off. I suppose one could say that it's important to generalise it outside of tugs of war, but it's not _that_ important -- as I'm sure students are capable of understanding that we're not actually setting up tugs of war each time, and so they understand it's just an analogy.

Idk

**Comment by towerofnix** [comment]
Fair enough, I guess. Sorry for making a long post without a point. I'm just... not entirely sure what the point of *this* - the project - is. So I figured I'd give an example and see your thoughts on it.

"Free maths learning" isn't enough to tell me what you have in mind, at all. I don't have any context.

**Comment by joker314** [comment]
Eh, you're right. I think what you wrote is exactly the sort of thing that could be in the site. You have a well-written block of text, followed by an interactive tug of war machine, followed by some fun questions that require actual creative problem solving.

To be honest, I _don't_ have anything in mind, and I'm very open to change from what we're discussing here if you want
